### PR TITLE
UTF-8

### PR DESCRIPTION
**Pull request by telemaxx**
and if now marker recourse found, a standard icon is created.

### CODE REVIEW

**Comment by telemaxx** [comment]
#237

**Comment by devemux86** [comment]
Thing is to post 1 commit with the correct encoding. That was the problem, cannot merge them with different encodings, git goes crazy..

**Comment by telemaxx** [comment]
> Thing is to post 1 commit with the correct encoding. That was the problem, cannot merge them with different encodings, git goes crazy..

was the problem, or is still the problem with this new pr?
if i open the java file with notepad++, n++ says also utf-8

i upload the java file via webui to my branch, is that the problem?

**Comment by devemux86** [comment]
The idea of the new PR was the convenience to have all commits squashed into one before posting it.
Or else can try squash them locally in your environment, there are instructions in [Contributing guide](https://github.com/mapsforge/vtm/blob/master/.github/CONTRIBUTING.md).

The final commit maybe correct, but the previous one is wrong and causes troubles in merging.
And really don't want to use GitHub automatic buttons for such things, prefer the safety of local git.

**Comment by devemux86** [comment]
> i upload the java file via webui to my branch, is that the problem?

Do you use Eclipse?
Usually a common workflow is to fork a repository, clone your fork, open it in IDE, create a feature branch, make your changes, commit them locally, push commits in fork, then post the PR.
When commit locally is best to squash commits into one final with all changes together.

**Comment by telemaxx** [comment]
> Do you use Eclipse?
Yes
> Usually a common workflow is to fork a repository, clone your fork, open it in IDE, create a feature branch, make your changes, commit them locally, push commits in fork, then post the PR.

Normally i do it this way, currently i work on a different project (mytourbook)
So the eclipse workspace is filled up a lot. for testing purposes i have parallel a little project:
https://github.com/telemaxx/vtm_with_gradle
here i commit/push my little tests apps

i dont have mapsforge/vtm  in my ide.
i have forked mapsforge/vtm via webui and uploaded my single java file to that fork and finally send  the PR to you.

> When commit locally is best to squash commits into one final with all changes together.

for me github is rocket science... i can do some simple things. But i dont really know, what i do.

**Comment by devemux86** [comment]
Ok let me try again if can remedy that... :slightly_smiling_face:

**Comment by devemux86** [comment]
Thanks, merged via 4162d9b4881c1007ed20d3fdf8e54083b14cea9d.

**Comment by telemaxx** [comment]
53645387-0e2ad000-3c39-11e9-8ebb-357ef1f1615f

**Comment by carlos-mg89** [comment]
I'm curious with the performance of this method when adding thousands of markers.

Yesterday I was doing some tests, and it seems a bit slow.
